tests/test_alias.rs

```diff
@@ -47,6 +47,7 @@ fn test_alias_calls_unknown_command() {
     "#,
     );
     let stderr = test_env.jj_cmd_cli_error(&repo_path, &["foo"]);
+    let stderr = stderr.replace(".exe", "");
```

Comment:
This probably makes sense in `normalize_output()`? Moving the similar calls to `.replace()` from `test_untrack_command.rs` and `test_global_opts.rs` to there would be a nice little cleanup.